deploy/README.md

```diff
@@ -19,7 +19,7 @@ Install the operator restricted to a single namespace.
 helm install elastic-operator-crds ./eck-operator/charts/eck-operator-crds
 
 # This step can be done by any user with full access to the my-namespace namespace.
-helm install elastic-operator eck-operator -n my-namespace --create-namespace \
+helm install elastic-operator elastic/eck-operator -n my-namespace --create-namespace \
```

Comment:
My feeling is that this would be inconsistent with the command line just above which refers to a directory:
> helm install elastic-operator-crds ./eck-operator/charts/eck-operator-crds

I think that with this change, we would mix up the instructions to install the chart from:
* a local copy of this ECK repo
* the official repository

Let's maybe clarify that these instructions are meant to be used to deploy the charts from this directory:

```diff
diff --git a/deploy/README.md b/deploy/README.md
index 0422afa40..e3d99bf6d 100644
--- a/deploy/README.md
+++ b/deploy/README.md
@@ -4,6 +4,8 @@
 
 This directory contains the Helm chart for deploying the ECK operator, and charts for deploying any resource in the Elastic Stack individually, or as a group.
 
+The instructions below are intended to deploy the Helm charts from a local copy of this repository. Refer to https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-install-helm.html to install the charts from the official repository.
+
 ## ECK Operator Helm Chart Usage
 
 Install the CRDs and deploy the operator with cluster-wide permissions to manage all namespaces.
```

Reply:
I absolutely see your point @barkbay . Maybe update to show this is a directory as well as adding the note?

```
helm install elastic-operator ./eck-operator -n my-namespace --create-namespace ...
```

Reply:
@naemono 👍 good point!

@framsouza sorry, it seems that what we would expect in this PR is maybe a bit more "advanced" than your initial proposal 🙃 Your PR is nevertheless valuable in the sense that it revealed something that could be misleading. Do you feel comfortable taking it over from here?